doc/manual/50-feature-details.en.md

```diff
@@ -13,24 +13,22 @@ common patterns.
 
 ### Incremental backup
 
-Barman implements **file-level incremental backup**. Incremental
-backup is a type of full periodic backup which only saves data changes
+Incremental backup is a type of full periodic backup which only saves data changes
```

Comment:
mmm, I would avoid using the "full" world.. as is sorta true when you use hard links with rsync but is not true when talking of incremental.

I think that we should reword this:
```
Incremental backup is a type of full periodic backup which only saves data changes
from the latest full backup available in the catalog for a specific
PostgreSQL server.
```
 in:
```
Incremental backup is a special type of backup which uses an already existing full backup as reference
for copying only necessary data changes from the PostgreSQL server.
```
@gustabowill @barthisrael

Reply:
Agree! I found that a little strange too at first glance too, but this is how it is in the current docs, so I just kept it.

Reply:
Updated this.

Reply:
Looks good.